openvm/tests/optimizer.rs

```diff
@@ -22,7 +22,7 @@ fn load_machine_cbor() {
             machine.bus_interactions.len(),
             machine.constraints.len()
         ],
-        [23838, 13167, 22998]
+        [27194, 13167, 27689]
```

Comment:
I updated the snapshot so that it also contains the constraints to fix the PC, because otherwise the assertion fails that all PC lookups are removed.